src/main/webapp/CollapsiblePanel/SampleDataCohortsCollapsibleHeader.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import { button, hh, span, div, i, a } from 'react-hyperscript-helpers';
+import { handleRedirectToConsentGroup, isEmpty } from '../util/Utils';
+import '../components/Btn.css';
+
+const POINTER = {
```

Comment:
nit, naming. Would read better if this were a styles object with pointer being one of the objects in it.
```suggestion
const styles = {
   pointer: { ... }
```